### DISCUSSION TITLE
哈佛 CS50 Python人工智能入门 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#48] [Announcements]
# 哈佛 CS50 Python人工智能入门 | HackWay技术学习路线

课程名称： Structure and Interpretation of Computer Programs

https://hackway.org/docs/AI/ai/cs50

**Comment by Evoque** [comment]
英文课程名错了，应该是: "Introduction to Artificial Intelligence with Python"

**Comment by Evoque** [comment]
想问下是否真的想好好经营这个网站， 里面的细节错误真的是不少
